server/statshouse/add_metrics_batch.h

```diff
@@ -0,0 +1,34 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2021 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace statshouse {
+
+struct tl_query_payload {
+  int *buffer;
+  int len;
+};
+
+struct StatsHouseMetric {
```

Comment:
this struct is inside namespace, whereas StatsHouseClient is not; is it on purpose?

Reply:
fixed